repository-data/application/src/main/resources/hcm-config/configuration/modules/ipfilter.yaml

```diff
@@ -15,13 +15,25 @@ definitions:
         - 81.21.138.121
         - 2001:4cb8:29d:1::/64
         cache-enabled: true
-        enabled: false
+        enabled: true
         forwarded-for-header: x-original-forwarded-for
         forwarded-host-header: x-forwarded-host
         hostnames:
-        - dev.nhsd.io
-        - cms-dev.nhsd.io
+        - cs.nhsdigital.bloomreach.cloud
         - development.nhsdigital.bloomreach.cloud
+        - tst.nhsdigital.bloomreach.cloud
+        - training.nhsdigital.bloomreach.cloud
+        - uat.nhsdigital.bloomreach.cloud
+        - cms-cs2.nhsd.io
+        - cms-development.nhsd.io
+        - cms-training2.nhsd.io
+        - cms-uat2.nhsd.io
+        - cs2.nhsd.io
+        - development.nhsd.io
+        - training2.nhsd.io
+        - uat2.nhsd.io
+        - cms-test.nhsd.io
+        - test.nhsd.io
```

Comment:
For readability, it would be better to group the CMS address with the corresponding SITE address. For example putting these together:

- cms-uat2.nhsd.io
- uat2.nhsd.io

Like you have with the test env.